doc/release_notes/release_notes_2.6.rst

```diff
@@ -80,8 +80,38 @@ Fixed Issues Details
 
 .. comment example item
    - :acrn-issue:`5626` - [CFL][industry] Host Call Trace once detected
-
+- :acrn-issue: '6012' -  [Mainline][PTCM] [ConfigTool]Obsolete terms cleanup for SSRAM
```

Comment:
The format of these issues is wrong (see the comment)

```suggestion
- :acrn-issue:`6012` -  [Mainline][PTCM] [ConfigTool]Obsolete terms cleanup for SSRAM
```

Reply:
@dbkinder Thanks!
Done!